<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>cframe</class>
 <widget class="QMainWindow" name="cframe">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1596</width>
    <height>834</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>cframe</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QStackedWidget" name="stackedWidget">
    <property name="geometry">
     <rect>
      <x>-10</x>
      <y>-10</y>
      <width>1581</width>
      <height>871</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true"/>
    </property>
    <property name="currentIndex">
     <number>2</number>
    </property>
    <widget class="QWidget" name="login">
     <property name="styleSheet">
      <string notr="true">#login {
    background-image: url(:/1/Imagenes/Fondo apple.webp);
    background-repeat: no-repeat;
    background-position: center;
    background-attachment: fixed;
}</string>
     </property>
     <widget class="QPushButton" name="pushButton">
      <property name="geometry">
       <rect>
        <x>750</x>
        <y>680</y>
        <width>83</width>
        <height>29</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>10</pointsize>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string>Entrar</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="lineEdit">
      <property name="geometry">
       <rect>
        <x>860</x>
        <y>580</y>
        <width>221</width>
        <height>28</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>10</pointsize>
       </font>
      </property>
     </widget>
     <widget class="QLineEdit" name="lineEdit_2">
      <property name="geometry">
       <rect>
        <x>500</x>
        <y>580</y>
        <width>221</width>
        <height>28</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>10</pointsize>
       </font>
      </property>
     </widget>
     <widget class="QLabel" name="label">
      <property name="geometry">
       <rect>
        <x>500</x>
        <y>560</y>
        <width>201</width>
        <height>21</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>10</pointsize>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string>Usuario:</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_2">
      <property name="geometry">
       <rect>
        <x>860</x>
        <y>560</y>
        <width>161</width>
        <height>21</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>10</pointsize>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string>Contraseña:</string>
      </property>
     </widget>
     <widget class="QPushButton" name="btnEnlaceLogin">
      <property name="geometry">
       <rect>
        <x>1310</x>
        <y>750</y>
        <width>231</width>
        <height>29</height>
       </rect>
      </property>
      <property name="text">
       <string>¿No tienes cuenta? Crea una aquí</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="page">
     <property name="styleSheet">
      <string notr="true"/>
     </property>
     <widget class="QLabel" name="label_3">
      <property name="geometry">
       <rect>
        <x>770</x>
        <y>680</y>
        <width>41</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string>Crear</string>
      </property>
     </widget>
     <widget class="Line" name="line">
      <property name="geometry">
       <rect>
        <x>690</x>
        <y>-100</y>
        <width>181</width>
        <height>791</height>
       </rect>
      </property>
      <property name="orientation">
       <enum>Qt::Orientation::Vertical</enum>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="page_3">
     <widget class="QSlider" name="volumen">
      <property name="geometry">
       <rect>
        <x>560</x>
        <y>380</y>
        <width>160</width>
        <height>18</height>
       </rect>
      </property>
      <property name="value">
       <number>20</number>
      </property>
      <property name="orientation">
       <enum>Qt::Orientation::Horizontal</enum>
      </property>
     </widget>
     <widget class="QSlider" name="sliderProgreso">
      <property name="geometry">
       <rect>
        <x>580</x>
        <y>230</y>
        <width>271</width>
        <height>16</height>
       </rect>
      </property>
      <property name="orientation">
       <enum>Qt::Orientation::Horizontal</enum>
      </property>
     </widget>
     <widget class="QLabel" name="labelTiempo">
      <property name="geometry">
       <rect>
        <x>860</x>
        <y>230</y>
        <width>101</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
     <widget class="QPushButton" name="mute">
      <property name="geometry">
       <rect>
        <x>430</x>
        <y>370</y>
        <width>83</width>
        <height>29</height>
       </rect>
      </property>
      <property name="text">
       <string>Mute</string>
      </property>
     </widget>
     <widget class="QPushButton" name="play">
      <property name="geometry">
       <rect>
        <x>440</x>
        <y>220</y>
        <width>83</width>
        <height>29</height>
       </rect>
      </property>
      <property name="text">
       <string>play</string>
      </property>
     </widget>
     <widget class="QPushButton" name="abrir">
      <property name="geometry">
       <rect>
        <x>440</x>
        <y>180</y>
        <width>83</width>
        <height>29</height>
       </rect>
      </property>
      <property name="text">
       <string>Abrir</string>
      </property>
     </widget>
     <widget class="QPushButton" name="pause">
      <property name="geometry">
       <rect>
        <x>440</x>
        <y>260</y>
        <width>83</width>
        <height>29</height>
       </rect>
      </property>
      <property name="text">
       <string>pause</string>
      </property>
     </widget>
     <widget class="QPushButton" name="stop">
      <property name="geometry">
       <rect>
        <x>440</x>
        <y>300</y>
        <width>83</width>
        <height>29</height>
       </rect>
      </property>
      <property name="text">
       <string>estop</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="page_4"/>
    <widget class="QWidget" name="page_crear"/>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1596</width>
     <height>25</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
